Nighttime-Operations/Observatory/DIMM/ops.rst

```diff
@@ -0,0 +1,194 @@
+.. |author| replace:: *isotuela*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *none*
+
+.. _DIMM-Ops:
+
+######################################
+DIMM Startup, Shutdown and Monitoring
+######################################
+
+The DIMM automatically performs seeing measurements from dusk until dawn. Limited support from the user is expected. 
```

Comment:
The "Limited support from the user is expected" sentence should be on a new line.

Reply:
👍